style.css

```diff
@@ -74,6 +74,151 @@ a {
   padding: 10px 5px;
 }
 
+/**  mobile  popup */
+
+.splash {
+  position: fixed;
+  backdrop-filter: blur(4px);
+  background-color: rgba(193, 199, 208, 0.5);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 24px;
+}
+
+.splash ul.due-time {
+  display: flex;
+  padding-left: 0;
+}
+
+.popup {
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  flex-basis: 100%;
+  display: flex;
+  padding: 2.4em 2.4em 6.6em  2.4em;
+  margin: 2% auto;
+  border-radius: 30px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  background-color: white;
+  height: 94%;
+  overflow-y: auto;
+  flex-direction: column;
+}
```

Comment:
- [ ] Good job implementing the popup windows :+1: However there is too much space between the popup's image and the its text (see image) compared to the Figma design. Kindly let's stick to the Figma design as much as possible.

### Your Design :point_down: 
![Screenshot from 2023-04-22 18-27-14](https://user-images.githubusercontent.com/91142432/233800986-601b71df-9cff-422a-8143-7fa0d1e93cf8.png)

### Figma's Design :point_down: 
![Screenshot from 2023-04-12 15-24-10](https://user-images.githubusercontent.com/91142432/233801001-116edbd7-06b9-4447-8c7e-21d1bbff85bb.png)